ceph_ansible.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- hosts: ceph_mon
```

Comment:
@fbalak I'm thinking about using dependencies, but I'm not sure if you can use anything else than role as dependency and execute it on different host, than the main role.